src/github.com/matrix-org/dendrite/clientapi/sync/syncapi/syncapi.go

```diff
@@ -0,0 +1,112 @@
+package syncapi
+
+import (
+	"strconv"
+
+	"github.com/matrix-org/gomatrixserverlib"
+)
+
+// StreamPosition represents the offset in the sync stream a client is at.
+type StreamPosition int64
+
+// String implements the Stringer interface.
+func (sp StreamPosition) String() string {
+	return strconv.FormatInt(int64(sp), 10)
+}
+
+// Response represents a /sync API response. See https://matrix.org/docs/spec/client_server/r0.2.0.html#get-matrix-client-r0-sync
```

Comment:
http://matrix.org/docs/spec/client_server/latest.html might be better than hardcoding 0.2.0 ? \<shrug>